packages/catalog-generator/scripts/build-catalogs.mjs

```diff
@@ -14,14 +14,14 @@ const require = createRequire(import.meta.url);
  **/
 const CATALOGS = {
   // https://repo1.maven.org/maven2/org/apache/camel/camel/
-  Main: ['4.8.1', '4.4.4', '4.4.0.redhat-00046', '4.8.0.redhat-00017'],
+  Main: ['4.9.0', '4.4.4', '4.4.0.redhat-00046', '4.8.0.redhat-00017'],
   // https://repo1.maven.org/maven2/org/apache/camel/quarkus/camel-quarkus/
   Quarkus: ['3.16.0', '3.8.3', '3.8.0.redhat-00018', '3.15.0.redhat-00007'],
   // https://repo1.maven.org/maven2/org/apache/camel/springboot/spring-boot/
-  SpringBoot: ['4.8.1', '4.4.4', '4.4.0.redhat-00039', '4.8.0.redhat-00022'],
+  SpringBoot: ['4.9.0', '4.4.4', '4.4.0.redhat-00039', '4.8.0.redhat-00022'],
```

Comment:
4.8.1 as LTS should stay and only be upgraded to 4.8.2. That also applies to Main